diff --git a/src/Repositories/ClientRepository.php b/src/Repositories/ClientRepository.php
index ec92d54d..3177c9a5 100644
--- a/src/Repositories/ClientRepository.php
+++ b/src/Repositories/ClientRepository.php
@@ -63,7 +63,7 @@ public function validateClient($client_identifier, $client_secret, $grant_type)
       return $this->passwordChecker->check($client_secret, $secret);
     }
 
-    return FALSE;
+    return TRUE;
   }
 
 }
